src/components/Message/MessageSimple/MessageStatus.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components/native';
 
 import Avatar from '../../Avatar/Avatar';
 
 import { useChatContext } from '../../../contexts/chatContext/ChatContext';
-import iconDeliveredUnseen from '../../../images/icons/delivered_unseen.png';
-import loadingGif from '../../../images/loading.gif';
+import { styled } from '../../../styles/styledComponents';
+
+import type { ForwardedMessageProps } from './MessageContent';
```

Comment:
I'd order this higher up based on it being `./` instead of `../../../`

Reply:
the way i've been doing it is: external imports > external type imports > local imports > local type imports, totally open to rearranging if we need